doc/source/ray-core/scheduling/placement-group.rst

```diff
@@ -309,19 +309,13 @@ You can schedule actors or tasks to a placement group using
 
   When you use an actor with a placement group, always specify ``num_cpus``.
 
-  When you don't specify (e.g., ``num_cpus=0``), a placement group option is ignored,
-  and the task and actor don't use the reserved resources.
-  
-  Note that by default (with no arguments to ``ray.remote``),
+  By default (with no arguments to ``ray.remote``), a Ray actor requires 1 CPU when it is scheduled. But after it is created, it occupies 0 CPUs.
 
-  - Ray task requires 1 CPU
-  - Ray actor requires 1 CPU when it is scheduled. But after it is created, it occupies 0 CPU.
+  When scheduling an actor with no resource requirements and a placement group, the placement group gets created (since the actor requires 1 CPU to be scheduled).
```

Comment:
This is also a bit unclear; does the placement group need to specify a bundle with {"CPU": 1} for the actor to be scheduled or is it possible to specify an empty bundle?